src/main/java/com/answerdigital/answerking/controller/CategoryController.java

```diff
@@ -45,14 +45,15 @@ public CategoryController(final CategoryService categoryService) {
     public ResponseEntity<Category> addCategory(@Valid @RequestBody final AddCategoryRequest categoryRequest,
                                                 final Errors errors) {
         return new ResponseEntity<>(categoryService.addCategory(categoryRequest),
-                                                        errors.hasErrors() ? HttpStatus.BAD_REQUEST : HttpStatus.OK);
+                                                        errors.hasErrors() ? HttpStatus.BAD_REQUEST : HttpStatus.CREATED);
     }
 
     @Operation(summary = "Get all categories.")
     @ApiResponses(value = {
         @ApiResponse(responseCode = "200", description = "Found the the list of categories",
             content = { @Content(mediaType = "application/json", schema = @Schema(implementation = Category.class)) })
     })
+
```

Comment:
I feel these should be connected as the @Operation is used for swagger documentation on the getAllCategories() endpoint, breaking the two up makes it look like a separate method

Reply:
Good eye! Didn't notice that- agreed!